spatialdata/_core/data_extent.py

```diff
@@ -0,0 +1,35 @@
+"""This file contains functions to compute the bounding box describing the extent of a spatial element,
```

Comment:
why this is not in the bounding box related module? I would put it there.

Reply:
this and other functions that could populate this file are not used for spatial queries so I would not put them in `_spatial_query.py`. The complexity of that file increase when implementing non-bounding box queries, so I would keep this code in another place.